src/Microsoft.AspNet.Mvc.Core/ControllerActionDescriptorBuilder.cs

```diff
@@ -299,6 +300,22 @@ private static ParameterDescriptor CreateParameterDescriptor(ParameterModel para
             }
         }
 
+        private static void AddProperties(
+            ControllerActionDescriptor actionDescriptor,
+            ActionModel action,
+            ControllerModel controller)
+        {
+            foreach (var item in controller.Properties)
```

Comment:
Discussed with @rynowak about this, we couldn't come up with any use cases for it.